app/controllers/concerns/blacklight/default_component_configuration.rb

```diff
@@ -64,39 +65,26 @@ def add_show_tools_partial name, opts = {}
         end
       end
 
-      ##
       # Add a tool to be displayed for each document in the search results.
-      # @param partial [String] the name of the document partial
-      # @param opts [Hash]
-      # @option opts [Symbol,Proc] :if render this action if the method identified by the symbol or the proc evaluates to true.
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      # @option opts [Symbol,Proc] :unless render this action unless the method identified by the symbol or the proc evaluates to true
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      def add_results_document_tool name, opts = {}
+      # @see #add_nav_action for params
```

Comment:
so personally i think any method of DRY in the docs is better than manual duplication. Whether see or a macro. I suspect the main advantage of see is just that it's easy to implement and easy for future comment-maintainers to understand without having to look up how to do macros and make it so.